OpenRA.Game/Traits/World/ActorMap.cs

```diff
@@ -47,12 +47,12 @@ class CellTrigger
 			public readonly CPos[] Footprint;
 			public bool Dirty;
 
-			readonly Action<Actor> onActorEntered;
-			readonly Action<Actor> onActorExited;
+			readonly Action<Tuple<Actor, Player>> onActorEntered;
+			readonly Action<Tuple<Actor, Player>> onActorExited;
```

Comment:
Alright, but then let's agree on an order of execution as well before I run another round at here. So currently it's `OnEntered` then `OnExited`, what I'd change `OnExited` then `OnEntered` tbh. Make `OnActorChanged` the last?